components/typography.css

```diff
@@ -1 +1,85 @@
-@import url(/stylesheets/main.css);;
\ No newline at end of file
+@import url(/stylesheets/main.css);
+@import url("https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/an-old-hope.min.css");
+
+/* font-sizes */
+.font-xxl {
+  font-size: 48px;
```

Comment:
You can use relative units(like rems) instead of px

Reply:
will make those changes while making library responsive